external/meshOptimizer/LICENSE.md

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Can you add external/meshOptimizer to the exceptions paths in scripts/astyle.sh?